### ISSUE TITLE
Upgrade to 4.3.2 breaks imports

### ISSUE THREAD

**Issue by dschuessler** [metadevpro/openapi3-ts#137]
After upgrading from version 4.3.1 to version 4.3.2, imports like:

```
import { ServerObject } from 'openapi3-ts/oas31'
```

fail with the following error

```
Cannot find module 'openapi3-ts/oas31' or its corresponding type declarations.
  There are types at '/path/to/my/project/node_modules/openapi3-ts/dist/oas31.d.ts', but this result could not be resolved under your current 'moduleResolution' setting.
Consider updating to 'node16', 'nodenext' or 'bundler'.
```

Setting the 'moduleResolution' setting from `node` to `node16`, `nodenext` or `bundler` is not an option because it causes errors with other packages.

Node version: 20.14.0

**Comment by pjmolina** [comment]
Should be fixed with 4.3.3. Can you confirm @dschuessler  ?

**Comment by dschuessler** [comment]
It's fixed. Thank you very much!
